packages/diagram/src/module/diagrams/content/uml/classifier/sections.ts

```diff
@@ -0,0 +1,50 @@
+import { InterpreterModule, parse } from "@hylimo/core";
+
+/**
+ * Module providing the sections content handler
+ */
+export const sectionsModule = InterpreterModule.create(
+    "uml/classifier/sections",
+    [],
+    [],
+    [
+        ...parse(
+            `
+                scope.internal.sectionsContentHandler = [
+                    {
+                        this.scope = args.scope
+                        scope.sections = list()
+                        scope.section = listWrapper {
+                            sectionIndex = args.section
+                            newSection = it
+                            if(sectionIndex == null) {
+                                scope.sections += newSection
+                            } {
+                                while { scope.sections.length <= sectionIndex } {
```

Comment:
Wait, `if` is just a function with an implicit `else` as well?

Reply:
yes